### DISCUSSION TITLE
unable to convert a jpeg image of size 181mb or of higher size to webp format

### DISCUSSION THREAD

**Discussion by Madhupatel08** [libvips/libvips#2839] [Q&A]
![image](https://user-images.githubusercontent.com/54492585/171940708-c67ba5a1-7640-48ad-9842-f345afae4133.png)
i used this, an error comes saying "unable to encode", same happens with imageimagic " 
![image](https://user-images.githubusercontent.com/54492585/171940996-924ad871-f3ad-4ded-bbaa-7f81c291b070.png)
"

**Comment by jcupitt** [comment]
Hi, I would guess your image is too large. webp is limited by design to 16383 x 16383 pixels.

Google promised a tiled image mode which would fix this, but they never got around to it. I think they abandoned webp development a while ago, and it's now mostly dead.

Hopefully more modern and better performing formats like jxl will take its place.
